MacaoProject/Macao/GamePlaceForm.cs

```diff
@@ -126,11 +170,102 @@ private void InitialDeck_MouseClick(object sender, MouseEventArgs e)
                 DiscardDeck.Image = RemainingCards[0].Image;
                 RemainingCards.RemoveAt(0);
             }
-            if(RemainingCards.Count ==0)
+            if (RemainingCards.Count == 0)
             {
                 InitialDeck.Image = null;
                 MessageBox.Show("There are no more cards");
             }
         }
+
+        private void ClickOnACard(PictureBox pictureCard, Card card)
+        {
+            if(card.Validation(card, newGame.StartCard) == true && pictureCard.Image != null)
```

Comment:
why the method Validation sends as the first parameter the card? you already are in the card class basically you are sending yourself as a parameter. The start card now it is not a start card anymore. it sound more like the top card or soemthing